pkg/deployments/tasks/goerli-tokens-factories/index.js

```diff
@@ -0,0 +1,547 @@
+        rlvFactory,
+        periphery,
+        adapterAddress,
+        stakeAddress,
+        balancerVault,
+        t,
+      });
+    } else {
+      // sponsor Series
+      for (let m of maturities) {
+        console.log(
+          `  * Sponsor Series for ${await adapterContract.symbol()} @ ${balancerVault} with maturity ${m}`,
+        );
+        await periphery.callStatic.sponsorSeries(adapterAddress, m, false);
+        await (await periphery.sponsorSeries(adapterAddress, m, false)).wait();
+      }
+    }
+
+    console.log("\n-------------------------------------------------------");
+  }
+});
+
+task("goerli-deploy-tokens", "Deploy tokens").setAction(async args => {
+  let { deploy, deployerSigner, data } = args;
+
+  let deployedTokens = [];
+  console.log("\n-------------------------------------------------------");
+  const { tokens, stake, reward } = data;
+  console.log("\nDeploy Tokens");
+  for (const token of tokens) {
+    for (const type in token) {
+      console.log("\n-------------------------------------------------------");
+      if (!token.underlying) throw Error("Missing underlying");
+      let underlying;
+      if (["target", "target4626"].includes(type)) {
+        underlying = token.underlying?.address || deployedTokens[token.underlying.symbol].address;
+      }
+      deployedTokens[token[type].symbol] = await deployToken(token[type], underlying, deploy, deployerSigner);
+      // ERC4626 tokens are not mintable, we would need to mint underlying and then deposit
+      if (type !== "target4626") {
+        console.log(`\n - Set relayer as trusted of ${token[type].symbol} so it can mint tokens`);
+        await trust(deployedTokens[token[type].symbol], OZ_RELAYER.get(CHAINS.GOERLI));
+      }
+    }
+  }
+  deployedTokens[stake.symbol] = await deployToken(stake, null, deploy, deployerSigner);
+  console.log(`\n - Set relayer as trusted of ${stake.symbol} so it can mint tokens`);
+  await trust(deployedTokens[stake.symbol], OZ_RELAYER.get(CHAINS.GOERLI));
+  deployedTokens[reward.symbol] = await deployToken(reward, null, deploy, deployerSigner);
+  console.log(`\n - Set relayer as trusted of ${stake.symbol} so it can mint tokens`);
+  await trust(deployedTokens[stake.symbol], OZ_RELAYER.get(CHAINS.GOERLI));
+
+  console.log("\n\n");
+  return deployedTokens;
+});
+
+task("goerli-rlv", "Create RLV").setAction(async args => {
+  let { deployer, relayerSigner, rlvFactory, periphery, adapterAddress, t, balancerVault, stakeAddress } =
+    args;
+
+  // create AutoRoller from OZ relayer
+  console.log(`\n- Deploy an AutoRoller for Ownable Adapter with target ${t.name} via AutoRollerFactory`);
+  const autoRollerFactory = await ethers.getContractAt("AutoRollerFactory", rlvFactory, relayerSigner);
+  let autoRollerAddress;
+  try {
+    autoRollerAddress = await autoRollerFactory.rollers(adapterAddress, 0);
+  } catch (e) {
+    console.log(`  ${adapterAddress} does not have an RLV. Creating one...`);
+  }
```

Comment:
can use a promise to simplify:

```suggestion
  const autoRollerAddress = await autoRollerFactory.rollers(adapterAddress, 0)
      .catch(e => console.log(`  ${adapterAddress} does not have an RLV. Creating one...`));
```

Reply:
oh, yeah, you've also wrote this on other review! i need to get used to that!